Toolset/palettes/tutorial/courses/First Run/tutorials/BMI Calculator/lessons/App Lesson.txt

```diff
@@ -1413,19 +1576,46 @@ step "Explain PreOpenCard Script"
 action
 	interlude
 	highlight line "on preOpenCard" of script editor for card "Chart"
+	go to step "Explain PreOpenCard Script II"
+end step
+
+step "Explain PreOpenCard Script II"
+	If the stack has a valid numeric cBMI custom property set on it, then
+	we highlight the spot on the chart where the value lies using the 
+	'hilitedCoordinates' property.
```

Comment:
I feel like all property and handler names should be formatted in the same way -- in some places you have them in quotes, and in others they're unquoted.